diff -Nru eplip-0.5.6.orig/Rules.make eplip-0.5.6/Rules.make
--- eplip-0.5.6.orig/Rules.make	Sun Mar 10 16:14:56 2002
+++ eplip-0.5.6/Rules.make	Sat Oct  5 14:29:54 2002
@@ -20,11 +20,6 @@
 # We need the configuration file, for CONFIG_SMP and possibly other stuff
 # (especiall for RISC platforms, where CFLAGS depends on the exact
 # processor being used).
-ifeq ($(KERNELDIR)/.config,$(wildcard $(KERNELDIR))/.config)
-	include $(KERNELDIR)/.config
-else
-	MESSAGE := $(shell echo "WARNING: no .config file in $(KERNELDIR)")
-endif
 
 # ARCH can be speficed on the comdline or env. too, and defaults to this arch
 # Unfortunately, we can't easily extract if from kernel configuration
@@ -83,10 +78,6 @@
 
 	CFLAGS +=  -Wall -Wstrict-prototypes  -Wwrite-strings -D__KERNEL__ -DMODULE -I$(INCLUDEDIR)
 
-ifdef CONFIG_SMP
-	CFLAGS += -D__SMP__ -DSMP
-endif
-
 # Prepend modversions.h if we're running with versioning.
 ifdef CONFIG_MODVERSIONS
 	CFLAGS += -DMODVERSIONS -include $(KERNELDIR)/include/linux/modversions.h
diff -Nru eplip-0.5.6.orig/Rules.make.smp eplip-0.5.6/Rules.make.smp
--- eplip-0.5.6.orig/Rules.make.smp	Sat Oct  5 14:26:41 2002
+++ eplip-0.5.6/Rules.make.smp	Sat Oct  5 14:30:42 2002
@@ -20,11 +20,6 @@
 # We need the configuration file, for CONFIG_SMP and possibly other stuff
 # (especiall for RISC platforms, where CFLAGS depends on the exact
 # processor being used).
-ifeq ($(KERNELDIR)/.config,$(wildcard $(KERNELDIR))/.config)
-	include $(KERNELDIR)/.config
-else
-	MESSAGE := $(shell echo "WARNING: no .config file in $(KERNELDIR)")
-endif
 
 # ARCH can be speficed on the comdline or env. too, and defaults to this arch
 # Unfortunately, we can't easily extract if from kernel configuration
@@ -83,9 +78,7 @@
 
 	CFLAGS +=  -Wall -Wstrict-prototypes  -Wwrite-strings -D__KERNEL__ -DMODULE -I$(INCLUDEDIR)
 
-ifdef CONFIG_SMP
-	CFLAGS += -D__SMP__ -DSMP
-endif
+	CFLAGS += -D__SMP__ -DSMP -DCONFIG_X86_LOCAL_APIC
 
 # Prepend modversions.h if we're running with versioning.
 ifdef CONFIG_MODVERSIONS
